app/code/Magento/Directory/Setup/Patch/Data/AddDataForUkraine.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Directory\Setup\Patch\Data;
+
+use Magento\Directory\Setup\DataInstaller;
+use Magento\Directory\Setup\DataInstallerFactory;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Add Ukraine Regions
+ */
+class AddDataForUkraine implements DataPatchInterface
+{
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+
+    /**
+     * @var DataInstallerFactory
+     */
+    private $dataInstallerFactory;
+
+    /**
+     * AddDataForUkraine constructor.
+     *
```

Comment:
Lets remove this comment, since doesn't provide any useful information.
```suggestion
```

Reply:
I'm not sure that it's really blocking approve PR. We have 80% of files with similar comment and I don't see any problems with this comment